recipe/meta.yaml

```diff
@@ -11,22 +11,23 @@ source:
 build:
   number: 0
   script:
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
     - rm "${SP_DIR}/pulp/solverdir/CoinMP.dll"  # [not win]
+  entry_points:
+    - pulptest = pulp.tests.run_tests:pulpTestAll
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
     - pip
     - python
+    - setuptools
+    - wheel
   run:
-    - amply >=0.1.2
-    # We use glpk on Windows because coincbc is only packaged on Unix, currently.
+    - python
+    - amply
```

Comment:
amply has been removed from the code https://github.com/coin-or/pulp/pull/403/files. There are no mentions in the source code already https://github.com/search?q=repo%3Acoin-or%2Fpulp+amply&type=pullrequests.
If we follow the conda-forge recipe can it be outdated?

Reply:
Yes, I think you are right. My bad, I did not check well enough.